src/components/Tag.js

```diff
@@ -110,30 +120,41 @@ const Tag = ({ color, disableMenu, slug, text, title, maxValue, ...props }) => {
               '?search[stringTraits][0][name]=' +
               title +
               '&search[stringTraits][0][values][0]=' +
-              text
+              value ?? text
           );
         }
       }
     );
-  }, [slug, text, title]);
+  }, [slug, text, value, title]);
+
+  const textWithUpdatedCase = keepTextLowerCase ? text : upperFirst(text);
+
+  const ButtonWrapper = ({ children }) =>
+    disableMenu ? (
+      children
+    ) : (
+      <ContextMenuButton
+        activeOpacity={0}
+        menuConfig={menuConfig}
+        {...(android ? { onPress: onPressAndroid } : {})}
+        enableContextMenu
+        isMenuPrimaryAction
+        onPressMenuItem={handlePressMenuItem}
+        useActionSheetFallback={false}
+        wrapNativeComponent={false}
+      >
+        {children}
+      </ContextMenuButton>
+    );
```

Comment:
Would such wrapper be okay? I think if parent of the children changes, this means re-create everything from scratch for React. Anyone has better idea or should that be fine?

Reply:
This is not happening in our app so I don't mind this. We have similar shit in many places. AFAIK in DS we have something for this